### PR TITLE
Add "Pick a reaction" header title to emoji dialog box and update display style.

### LINKED ISSUE #4993: Add "Pick a reaction" text to reaction dropdown on comment
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

Add the text "Pick a reaction" in reaction dropdown just like GitHub have :point_down: 
reaction_text

Our current reaction dropdown look :point_down: 
current_pl

Feel free to optimize UI of dropdown further too.

- [ ] 📝 **Update** You need to update these lines for implementing the change - https://github.com/publiclab/plots2/blob/master/app/views/notes/_comment.html.erb#L96-L106

Also, attach the screenshot of the updated view by going over to any research note/question on localhost and taking a screenshot of the comment posted over there

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by Shreets**
- Add 'Pick a reaction' header to emoji dialog box.
- Add margin/padding and flex display to emoji style. 
- Add hover effect to scale emojis.

Fixes #4993 

reaction-bar

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

@publiclab/reviewers Please review.
Thank you!

### CODE REVIEW

File: app/views/notes/_comment.html.erb
Comment (jywarren) [inline app/views/notes/_comment.html.erb:226]: wow, fancy!!! ❤️ 🤯

File: app/views/notes/_comment.html.erb
Comment (jywarren) [inline app/views/notes/_comment.html.erb:211]: Could we actually make a more specific css classname here? `.open` seems likely to collide with another stylesheet somewhere, someday... you know? What do you think?

**Review by jywarren — APPROVED** [review]
This looks great and happy to merge! One possible note to change the classname, but not required! Thank you so much!

**Comment by jywarren** [comment]
So, I think this is good to go, but can hold off a little if you'd like to change the classname -- your choice! Thanks!

File: app/views/notes/_comment.html.erb
Comment (sagarpreet-chadha) [reply app/views/notes/_comment.html.erb:211]: Correct we should!

**Review by sagarpreet-chadha — APPROVED** [review]
Wooow 🎉

**Comment by Shreets** [comment]
> So, I think this is good to go, but can hold off a little if you'd like to change the classname -- your choice! Thanks!


@jywarren @sagarpreet-chadha Thank you for pointing that out. I will make this change and update the PR right away.

File: app/views/notes/_comment.html.erb
Comment (Shreets) [reply app/views/notes/_comment.html.erb]: I have changed the class name from `.open` to `.emoji-container`.
Please review.
Thank you!

**Review by Sagarpreet — APPROVED** [review]
🎉

**Comment by jywarren** [comment]
Great work!!!!!!!!!! 🎉

**Pull request merged**
